src/main/webapp/app/exercises/shared/team/teams-import-dialog/teams-import-dialog.component.ts

```diff
@@ -40,11 +40,17 @@ export class TeamsImportDialogComponent implements OnInit, OnDestroy {
     readonly defaultImportStrategy: ImportStrategy = ImportStrategy.CREATE_ONLY;
 
     isImporting = false;
+    showImportFromExercise = true;
 
     // computed properties
     teamShortNamesAlreadyExistingInExercise: string[] = [];
     studentLoginsAlreadyExistingInExercise: string[] = [];
     sourceTeamsFreeOfConflicts: Team[] = [];
+    studentRegistrationNumbersAlreadyExistingInOtherTeams: string[] = [];
```

Comment:
we have quite a lot of string array properties here.
Can we handle this more elegant?